lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -16310,10 +16310,10 @@ msgid "Status type"
 msgstr "Type de status"
 
 msgid "Hard"
-msgstr "Dur"
+msgstr ""
 
 msgid "Soft"
-msgstr "Souple"
+msgstr ""
```

Comment:
I think this will add empty translation. We should comment them by adding a '#' at the beginning of each line

Reply:
if I comment the lines , I do not receive the labels , and they are displayed empty